DeviceAdministration/Web/Scripts/Views/Dashboard/TelemetryHistory.js

```diff
@@ -215,7 +228,7 @@
             height = $(targetControl).height();
             width = $(targetControl).width();
 
-            if (lastData) {
+            if (lastData && hasVisualBeenInitialized) {
```

Comment:
It seems like the visual is sometimes updated before it's created for the first time. This ensures that doesn't happen